### PR TITLE
eclipse.appengine.login --> .login

### LINKED ISSUE #396: Change ".eclipse.appengine.login" to ".eclipse.login" and remove all "appengine" naming in login
A task item in #342.

### PR DESCRIPTION
**Pull request by elharo**
@chanseokoh @abbiefarr Fix #396 since we'll have non appengine code soon that uses login

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.login.test/src/com/google/cloud/tools/eclipse/login/ui/LoginServiceUiTest.java [hunk 1]
```diff
@@ -32,6 +31,8 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+import com.google.cloud.tools.eclipse.login.ui.LoginServiceUi.LocalServerReceiverWrapper;
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.login.test/src/com/google/cloud/tools/eclipse/login/ui/LoginServiceUiTest.java]: Will you retain the original import order?

File: plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/CredentialHelper.java [hunk 2]
```diff
@@ -14,12 +14,13 @@
...
  */
 
-package com.google.cloud.tools.eclipse.appengine.login;
+package com.google.cloud.tools.eclipse.login;
 
 import java.util.HashMap;
 import java.util.Map;
 
 import com.google.api.client.auth.oauth2.Credential;
+import com.google.cloud.tools.eclipse.login.Constants;
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/CredentialHelper.java]: This won't be necessary. Might be the result of a timing issue when auto-generating `Constants.java`. Try reloading projects if you see any problem.

File: plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/GoogleLoginService.java [hunk 3]
```diff
@@ -14,10 +14,11 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.eclipse.appengine.login;
+package com.google.cloud.tools.eclipse.login;
 
 import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeRequestUrl;
-import com.google.cloud.tools.eclipse.appengine.login.ui.LoginServiceUi;
+import com.google.cloud.tools.eclipse.login.Constants;
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/GoogleLoginService.java]: Ditto. This won't be necessary.

File: plugins/com.google.cloud.tools.eclipse.test.logback/.project [hunk 4]
```diff
@@ -11,17 +11,17 @@
 			</arguments>
 		</buildCommand>
 		<buildCommand>
-			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<name>org.eclipse.pde.ManifestBuilder</name>
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.test.logback/.project]: This is a completely unrelated bundle. Shouldn't be in this PR.

File: plugins/com.google.cloud.tools.eclipse.login.test/src/com/google/cloud/tools/eclipse/login/ui/LoginServiceUiTest.java (hunk 1 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.login.test/src/com/google/cloud/tools/eclipse/login/ui/LoginServiceUiTest.java]: We haven't really set any rules on import order. I think we have a mix of old and new Google style for this. In any case, done.

File: plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/CredentialHelper.java (hunk 2 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/CredentialHelper.java]: done

File: plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/GoogleLoginService.java (hunk 3 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/GoogleLoginService.java]: done

File: plugins/com.google.cloud.tools.eclipse.test.logback/.project (hunk 4 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.test.logback/.project]: We probably shouldn't be checking these autogenerated files into the repo in the first place. They cause churn for no particular benefit. let me see if I can revert it.
